src/PodiumdAdapter.Web/Endpoints/ObjectenEndpoints/ObjectenEndpoints.cs

```diff
@@ -468,9 +468,16 @@ private static string HaalLaatsteVoorlopigeAntwoordOp(JsonNode? contactmoment)
                 && recentsteVoorlopigAntwoord["volledigeNaam"]?.GetValue<string>() is string volledigeNaam
                 && recentsteVoorlopigAntwoord["registratiedatum"]?.GetValue<string>() is string registratieDatum)
             {
-                var parsedDate = DateTime.Parse(registratieDatum);
-                var formattedDate = parsedDate.ToString("dd-MM-yyyy, HH:mm");
+
+                var dateTimeOffset = DateTimeOffset.Parse(registratieDatum);
+
+                // Convert to the time zone "+02:00" explicitly
+                var targetTimeZoneOffset = TimeSpan.FromHours(2);  // As your original time has "+02:00"
```

Comment:
klopt dat in de wintertijd ook? zit de offset in de info die wij binnenkrijgen en kunne  we die gebruiken? dan doen we altijd de juiste correctie ipv hard een 2 uur offset inbouwen